community/grafana/PKGBUILD

```diff
@@ -0,0 +1,101 @@
+# Maintainer: Sébastien "Seblu" Luttringer
+# Contributor: Carsten Feuls <archlinux@carstenfeuls.de>
+# Contributor: Gilles Hamel <hamelg at laposte dot net>
+
+pkgname=grafana
+pkgver=7.3.5
+pkgrel=1
+pkgdesc='Gorgeous metric viz, dashboards & editors for Graphite, InfluxDB & OpenTSDB'
+url='https://grafana.com/'
+arch=('x86_64')
+license=('Apache')
+depends=('glibc' 'freetype2' 'fontconfig' 'gsfonts')
+makedepends=('git' 'go' 'npm' 'grunt-cli' 'python' 'nodejs-lts-erbium')
```

Comment:
you will also need `python2` as a dep for `node-sass` and `iltorb`

Reply:
Are you sure about this? It's hard to test for me with a clean chroot on Raspberry Pi. But on x86_64 it compiles fine without python2 dependency.

Reply:
yes as i tried to build it today on a (mostly) fresh `aarm64`. for `iltorb` it threw an error where it trys to print in python2 syntax and fails. Also it says that the error can be ignored as it is an optional dependency. For `node-sass` it says that it searches for `/usr/bin/python2` and after that searches for `python` in the path and then fails again with a syntax error on a line that trys to use python2 syntax. After installing python2 and rerunning `makepkg -csi` it worked